exercises/grade-school/GradeSchoolTest.swift

```diff
@@ -2,84 +2,58 @@
     import XCTest
 #endif
 
-private extension XCTestCase {
-
-    // Workaround unit test liminitaiton on Optional Arrays/Sets
-    private func sameCollection<C: Collection where C.Iterator.Element == String>(_ result: C?, _ expected: [String]?) -> Bool {
-        guard let result = result, expected = expected where similar(result, expected) else { return false }
-        for (index, student) in result.enumerated() {
-            guard index < expected.count && expected.contains(student) else { return false }
-        }
-        return true
-    }
-    private func similar<C: Collection>(_ result: C, _ expected: [String]) -> Bool {
-        return result.count.toIntMax() == IntMax(expected.count)
-    }
-
-    func XCTAssertEqualCollection (_ collectionS: Set<String>?, _ collectionA: [String]? ) {
-        XCTAssert(sameCollection(collectionS, collectionA))
-    }
-
-    func XCTAssertEqualCollection (_ collectionA1: [String]?, _ collectionA2: [String]? ) {
-        XCTAssert(sameCollection(collectionA1, collectionA2))
-    }
```

Comment:
So there is an issue here with people trying to use a set instead of an array or the other way around that's why we added this. Do you think we can support both with out it?